test/performance/benchmarks/channel-imc/prod.config

```diff
@@ -0,0 +1,57 @@
+### Creating this benchmark:
+# mako create_benchmark test/performance/channel-imc/prod.config
+### Updating this benchmark:
+# mako update_benchmark test/performance/channel-imc/prod.config
+
+project_name: "Knative"
+benchmark_name: "IMC Channel Latency & Throughput"
+description: "Measure latency and throughput of channels."
+benchmark_key: '5683818216292352'
+
+# Human owners for manual benchmark adjustments.
+owner_list: "grantrodgers@google.com"
+owner_list: "chizhg@google.com"
+owner_list: "xiyue@google.com"
+owner_list: "gracegao@google.com"
+owner_list: "nachocano@google.com"
+owner_list: "cshou@google.com"
+
+# Only this robot should publish data to Mako for this key!
+owner_list: "mako-job@knative-performance.iam.gserviceaccount.com"
+
+# Define the name and type for x-axis of run charts
+input_value_info: {
+  value_key: "t"
+  label: "time"
+  type: TIMESTAMP
+}
+
+# Note: value_key is stored repeatedly and should be very short (ideally one or two characters).
+metric_info_list: {
+  value_key: "pl"
+  label: "publish-latency"
+}
+metric_info_list: {
+  value_key: "pe"
+  label: "publish-errors"
+}
+metric_info_list: {
+  value_key: "st"
+  label: "send-throughput"
+}
+metric_info_list: {
+  value_key: "dl"
+  label: "deliver-latency"
+}
+metric_info_list: {
+  value_key: "de"
+  label: "deliver-errors"
+}
+metric_info_list: {
+  value_key: "dt"
+  label: "deliver-throughput"
+}
+metric_info_list: {
+  value_key: "ft"
```

Comment:
This is incorrect. The current perf test uses `pet` and `det` instead of `ft`, see https://github.com/knative/eventing/blob/master/test/test_images/performance/prod.config